packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -121,10 +220,24 @@ const BlocksToolbar = () => {
           </Flex>
         </Toolbar.ToggleGroup>
         <Separator />
-        <Toolbar.ToggleGroup type="multiple" asChild>
+        <Toolbar.ToggleGroup type="single" asChild>
           <Flex gap={1}>
-            <Toolbar.ToggleItem value="test">test</Toolbar.ToggleItem>
-            <Toolbar.ToggleItem value="test2">test</Toolbar.ToggleItem>
+            <ListButton
+              label={{
+                id: 'components.Blocks.blocks.unorderedList',
+                defaultMessage: 'Unordered list',
```

Comment:
I wanted to match what's saved in the schema, and in the schema @markkaylor and I thought "ordered" and "ordered" were better because they make no UI assumptions. For example an unordered list can we rendered with squares or arrows, not necessarily bullets